agent/agent-bootstrap/src/main/java/com/microsoft/applicationinsights/agent/bootstrap/diagnostics/status/StatusFile.java

```diff
@@ -62,33 +65,16 @@ public class StatusFile {
   // visible for testing
   static final String FILE_EXTENSION = ".json";
 
-  // visible for testing
-  static final String SITE_LOGDIR_PROPERTY = "site.logdir";
-
   // visible for testing
   static final String HOME_ENV_VAR = "HOME";
 
   // visible for testing
-  static final String DEFAULT_HOME_DIR = ".";
-
-  // visible for testing
-  static final String DEFAULT_LOGDIR = "/LogFiles";
-
-  // visible for testing
-  static final String DEFAULT_APPLICATIONINSIGHTS_LOGDIR = "/ApplicationInsights";
-
-  // visible for testing
-  static final String STATUS_FILE_DIRECTORY = "/status";
```

Comment:
just checking, the status file doesn't go under `/status` directory on windows?

Reply:
good catch. i tested it again. windows requires this folder. fixed it.